Doc/library/sqlite3.rst

```diff
@@ -2269,8 +2339,67 @@ Explanation
 Transaction control
 ^^^^^^^^^^^^^^^^^^^
 
-The :mod:`!sqlite3` module does not adhere to the transaction handling recommended
-by :pep:`249`.
+.. _sqlite3-transaction-control-autocommit:
+
+Transaction control via the ``autocommit`` attribute
+""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+The recommended way of controlling transaction behaviour is through
+the :attr:`Connection.autocommit` attribute.
+This attribute should preferrably be set using the *autocommit* parameter
+of :func:`connect`.
+
+It is recommended to set *autocommit* to ``False``,
+implying :pep:`249`-compliant transaction control.
+This means:
+
+* :mod:`!sqlite3` ensures that a transaction is always open,
+  so :meth:`~Connection.commit` and :meth:`~Connection.rollback`
```

Comment:
```suggestion
  so :meth:`Connection.commit` and :meth:`Connection.rollback`
```

Seems better to be explicit these these are methods of this class, not, e.g., functions of the module